tf2onnx/rewriter/layer_normalization_rewriter.py

```diff
@@ -0,0 +1,122 @@
+# SPDX-License-Identifier: Apache-2.0
+
+
+"""
+tf2onnx.rewrite - Rewrites a specific pattern that appears in some models from the tf layer_norm contrib op
```

Comment:
We should have a more specific comment. This one does not say which optimization is done here.

Reply:
Fair point.